std/json.d

```diff
@@ -628,7 +655,7 @@ struct JSONValue
     /// Implicitly calls $(D toJSON) on this JSONValue.
     ///
     /// $(I options) can be used to tweak the conversion behavior.
-    string toString(in JSONOptions options = JSONOptions.none) const
+    string toString(in JSONOptions options = JSONOptions.none) const @safe
     {
         return toJSON(&this, false, options);
```

Comment:
It's marked `in`, which means the value is not allowed to escape the function. That means most of the un@safe abuses aren't allowed.

Beyond that, I would have changed it to `ref`, but that's a breaking change.